src/deploy/NVA_build/FrontendLib.Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM noobaa-builder
```

Comment:
this dockerfile is for the test only right ?

Reply:
Yes, as building the frontend dependencies needs tools such as node, npm, and some dependencies from the core, the easiest way to get it is by running docker built.
Noobaa-builder containers all the build needs, and on this Dockerfile (FrontendLib)  we are also running `npm install --production` on the core so we will have the modules required by the frontend.